matlab/lpo_mex.cpp

```diff
@@ -328,51 +292,28 @@ std::shared_ptr<SeedFunction> createSeed( const std::string & s ) {
     }
     return seed;
 }
+static std::string model;
 static void newProposal( MEX_ARGS ) {
     if( nlhs == 0 ) {
         mexErrMsgTxt("newProposal expected one return argument");
         return;
     }
-    // Create an empty unary
-    ProposalSettings prop_settings;
-    prop_settings.unaries.clear();
-    
-    // Add all settings
+
     for ( int i=0; i<nrhs; i++ ) {
         std::string c = toString( prhs[i] );
-        
+
         // Process the command
-        if (c == "max_iou") {
-            if( i+1>=nrhs || !mxIsNumeric(prhs[i+1]) )
-                mexErrMsgTxt("max_iou numeric argument required");
-            prop_settings.max_iou = mxGetScalar(prhs[i+1]);
-            i++;
-        }
-        else if (c == "seed") {
+        if (c == "model") {
```

Comment:
If the function just has a single model argument, we can skip the c=="model" part and give it the model path directly.